examples/krops/node.nix

```
@@ -0,0 +1,27 @@
+let
+  # FIXME
+  target = "root@NODE_SSH_HOST_OR_IP_ADDRESS_OR_HOST_NAME_HERE";
+
+  pkgs = import <nixpkgs> {};
+  nixBitcoinPkgs = import <nix-bitcoin> {};
+  source = nixBitcoinPkgs.krops.lib.evalSource [{
+    nixpkgs.file = {
+      path = toString <nixpkgs>;
+      useChecksum = true;
+    };
+    nix-bitcoin.file = {
+      path = toString <nix-bitcoin>;
+      useChecksum = true;
+    };
```

I think we should exclude the `.git` folder for people deploying from git repos
```suggestion
    filters = [
      {
        type = "exclude";
        pattern = ".git";
      }
    ];
```

---

Fixed.